### DISCUSSION TITLE
Kernel version: v1.28.2-3598+6e2789bbd58938-dirty causes installation to fail

### DISCUSSION THREAD

**Discussion by parseltongued** [okd-project/okd#2009] [General]
**Describe the bug**
Storage CNI installation tool expects Kernel version without +6e2789bbd58938-dirty but `oc --version` states Kernel version: v1.28.2-3598+6e2789bbd58938-dirty

**Version**
OKD/Openshift Version: 4.15.0-0.okd-2024-03-10-010116

How to remove +6e2789bbd58938-dirty?

**Comment by JaimeMagiera** [comment]
Hi,

We are not working on FCOS builds of OKD any more. Please see these documents...

https://okd.io/blog/2024/06/01/okd-future-statement
https://okd.io/blog/2024/07/30/okd-pre-release-testing

I'll convert this to a discussion in case anyone in the community can help. 

Many thanks,

Jaime
